tests/acceptance/features/bootstrap/GraphContext.php

```diff
@@ -96,11 +185,9 @@ public function userHasBeenEditedUsingTheGraphApi(
 	 */
 	public function adminHasRetrievedUserUsingTheGraphApi(string $user): void {
 		$user = $this->featureContext->getActualUsername($user);
-		try {
-			$userId = $this->featureContext->getAttributeOfCreatedUser($user, "id");
-		} catch (Exception $e) {
-			$userId = $user;
-		}
+		$userId = $this->featureContext->getAttributeOfCreatedUser($user, "id");
+		$userId = $userId ?: $user;
```

Comment:
null coalescing operator should be more readable here instead of ternary
```suggestion
		$userId = $userId ?? $user;
```